components/user-mgt/org.wso2.carbon.user.mgt/src/main/java/org/wso2/carbon/user/mgt/bulkimport/CSVUserBulkImport.java

```diff
@@ -49,91 +55,76 @@ public CSVUserBulkImport(BulkImportConfig config) {
     public void addUserList(UserStoreManager userStore) throws UserAdminException {
         CSVReader csvReader = new CSVReader(reader, ',', '"', 1);
         try {
-
-            String domain = config.getUserStoreDomain();
+            userStoreDomain = config.getUserStoreDomain();
             String[] line = csvReader.readNext();
             boolean isDuplicate = false;
             boolean fail = false;
-            boolean success = false;
-            String lastError = "UNKNOWN";
-            int successCount = 0;
-            int failCount = 0;
-            int duplicateCount = 0;
             while (line != null && line.length > 0) {
                 String userName = line[0];
 
                 int index;
                 index = userName.indexOf(CarbonConstants.DOMAIN_SEPARATOR);
                 if (index > 0) {
                     String domainFreeName = userName.substring(index + 1);
-                    userName = UserCoreUtil.addDomainToName(domainFreeName, domain);
+                    userName = UserCoreUtil.addDomainToName(domainFreeName, userStoreDomain);
                 } else {
-                    userName = UserCoreUtil.addDomainToName(userName, domain);
+                    userName = UserCoreUtil.addDomainToName(userName, userStoreDomain);
                 }
 
-                if (userName != null && userName.trim().length() > 0) {
+                if (StringUtils.isNotEmpty(userName) && userName.trim().length() > 0) {
```

Comment:
Shall we use `StringUtils.isNotBlank()` here? So we can cover the whole condition.